pkg/api/validation/validation.go

```diff
@@ -1921,6 +1921,10 @@ func ValidateSecurityContext(sc *api.SecurityContext) errs.ValidationErrorList {
 			allErrs = append(allErrs, errs.NewFieldInvalid("runAsUser", *sc.RunAsUser, "runAsUser cannot be negative"))
 		}
 	}
+
+	if sc.SELinuxOptions != nil && !capabilities.Get().EnableSELinuxIntegration {
+		allErrs = append(allErrs, errs.NewFieldForbidden("selinuxOptions", "Cannot specify selinuxOptions if SELinux integration is not enabled"))
```

Comment:
I'm not questioning if it belongs in both places.  If we want the verification then this is correct.  But I'm wondering if this is an error condition at all.  If I pass a pod with SELinuxOptions and the cluster is not using SELinux why should it fail?  Presumably my pod can still run fine